test/orchard/meta_test.clj

```diff
@@ -2,7 +2,9 @@
   (:require
    [orchard.meta :as m]
    [clojure.repl :as repl]
-   [clojure.test :refer :all]))
+   [clojure.test :refer :all]
+   [clojure.set :as set]
+   [clojure.spec.alpha :as s]))
```

Comment:
The spec code should be conditionally added, as we still support Clojure 1.8.

Reply:
yeah just saw the test matrix failure. I'll get to this tonight.